instrumentation/sinatra/lib/opentelemetry/instrumentation/sinatra/instrumentation.rb

```diff
@@ -51,19 +51,7 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
         end
 
         def install_middleware(app)
-          if config[:install_rack]
-            stability_opt_in = ENV.fetch('OTEL_SEMCONV_STABILITY_OPT_IN', '')
-            values = stability_opt_in.split(',').map(&:strip)
-
-            if values.include?('http/dup')
-              app.use(*OpenTelemetry::Instrumentation::Rack::Instrumentation.instance.middleware_args_dup)
-            elsif values.include?('http')
-              app.use(*OpenTelemetry::Instrumentation::Rack::Instrumentation.instance.middleware_args_stable)
-            else
-              app.use(*OpenTelemetry::Instrumentation::Rack::Instrumentation.instance.middleware_args)
-            end
-          end
-
+          app.use(*OpenTelemetry::Instrumentation::Rack::Instrumentation.instance.middleware_args) if config[:install_rack]
```

Comment:
This all looks so great. Thank you for working on this. 

Two Sinatra tests are using the direct stable/dup methods. It would be great to update those tests to better match the changes you've made by replacing `middleware_args_stable` and `middleware_args_dup` with `middleware_args`.

The files are:
- `sinatra/test/opentelemetry/instrumentation/sinatra_dup_http_test`
- `sinatra/test/opentelemetry/instrumentation/sinatra_stable_http_test`